PYME/experimental/_triangle_mesh.pyx

```diff
@@ -188,8 +188,8 @@ VERTEX_DTYPE2 = np.dtype([('position0', 'f4'),
 cdef extern from "triangle_mesh_utils.c":
     void _update_face_normals(np.int32_t *f_idxs, halfedge_t *halfedges, vertex_d *vertices, face_d *faces, signed int n_idxs)
     
-    void update_face_normal(int f_idx, halfedge_t *halfedges, vertex_d *vertices, face_d *faces)
-    void update_single_vertex_neighbours(int v_idx, halfedge_t *halfedges, vertex_d *vertices, face_d *faces)
+    void update_face_normal(np.int32_t f_idx, halfedge_t *halfedges, vertex_d *vertices, face_d *faces)
```

Comment:
This should be `int`, as it's used as an index into the array. Otherwise we force a cast when we access array elements. The fact that the indices are stored as `int32_t` within the structures to save memory is irrelevant.

Reply:
Fair enough.